pkcs12.go

```diff
@@ -66,12 +66,13 @@ type encryptedPrivateKeyInfo struct {
 func (i encryptedPrivateKeyInfo) GetAlgorithm() pkix.AlgorithmIdentifier { return i.AlgorithmIdentifier }
 func (i encryptedPrivateKeyInfo) GetData() []byte                        { return i.EncryptedData }
 
-// Error indicating that the input is in an unsupported format. The format may be allowed by the RFC's, but may not be implemented by this package.
+// UnsupportedFormat Error indicates that the input is in an unsupported format.
+// The format may be allowed by the RFC's, but may not be implemented by this package.
 type UnsupportedFormat string
```

Comment:
Usually these types have an Error suffix (`UnsupportedFormatError`), but I avoided changing your API and breaking the callers.